CHANGELOG.md

```diff
@@ -5,11 +5,10 @@ All notable changes to PowerModelsMCDC.jl will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
```

Comment:
Version information and date are supposed to be updated only by the maintainer in the same commit used for registering the new release. Prior to that, the [`Unreleased` section](https://keepachangelog.com/en/1.1.0/#effort) is meant to collect the committed changes. That way, the information reported in the changelog is kept consistent with the actual status of the code.